geode-core/src/main/java/org/apache/geode/internal/cache/PoolFactoryImpl.java

```diff
@@ -328,14 +329,16 @@ public void init(GatewaySender sender) {
    * @since GemFire 5.7
    */
   public Pool create(String name) throws CacheException {
+    InternalDistributedSystem distributedSystem = InternalDistributedSystem.getAnyInstance();
```

Comment:
Aww, I'm a little disappointed that we couldn't get rid of this entirely.

Reply:
Yes. This is an incremental step rather than the final solution.